CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file. See [standa
 
 ## [0.11.0](https://github.com/getditto/react-ditto/compare/v0.11.0-alpha.0...v0.11.0) (2023-03-22)
 
+### Unreleased
+
+### Changed
+
+- updated `create-react-app` example in Readme ([#59](https://github.com/getditto/react-ditto/pull/59))
+
```

Comment:
I don't think this PR warrants a changelog entry, i.e., it would be a `chore:` or `docs:` commit. Would you mind removing this?